src/components/search/index.tsx

```diff
@@ -0,0 +1,96 @@
+import Downshift from 'downshift';
```

Comment:
I'm trying to be better at being consistent at testing (😅), so it would be interesting to see if we can add some tests to this component

Reply:
😳

I should be too. Will get on it and add some. Adding it to the list.

Reply:
MIght need some inspiration on how to test.

Reply:
I think the cookie context tests can be used as inspiration of rendering and user interactions.

Reply:
Maybe also with SWR fallback provider (look at swr documentation) to return a result for `useAutocomplete`.

Reply:
Will have a look at that. I assume we want this isolated and not calling the API.

Reply:
Yeah. I think for the actual component test we can use fallback for swr to avoid doing actual request. I haven't actually tried it, but I'm pretty sure swr has some solutions for mocking data. If not it is possible to use some sort of HTTP mocking service like mwn (or what it is called)

Reply:
Looks like vitest has some mocking: https://vitest.dev/guide/mocking.html

Reply:
Yeah, but this is mocking of modules, not of HTTP results. Both are fine, but it depends on of what degree you want to test (unit vs integration)

Reply:
mwn (or something like that, I don't actually remember the module name) instead monkey patches `fetch` and returns faux results on HTTP requests. Same, same, but different.

Reply:
I see they mention MSW: https://vitest.dev/guide/mocking.html#requests

Reply:
that's it!